cfme/containers/image.py

```diff
@@ -152,6 +153,81 @@ def all(self):
                                            provider=provider or get_crud_by_name(ems_name)))
         return images
 
+    def check_compliance_multiple_images(self, image_entitys, wait_for_finish=True, timeout=240):
```

Comment:
__REQUIRED__
Please add docstring acording to [pep-0257](https://www.python.org/dev/peps/pep-0257/).

Reply:
Done , add arguments to docstring.
change attribute name wait_for_finish to check_on_entity